controllers/infrastructure/byomachine_controller_test.go

```diff
@@ -479,6 +480,26 @@ var _ = Describe("Controllers/ByomachineController", func() {
 					Severity: clusterv1.ConditionSeverityInfo,
 				}))
 			})
+
+			It("should mark BYOHostReady as False when is secret is not available", func() {
```

Comment:
```suggestion
			It("should mark BYOHostReady condition as False when the secret is not available", func() {
```

Reply:
when `InstallationSecret` is not available